plugins/woocommerce/includes/wc-formatting-functions.php

```diff
@@ -1004,9 +1007,12 @@ function wc_format_postcode( $postcode, $country ) {
 			$postcode = substr_replace( $postcode, ' ', 4, 0 );
 			break;
 		case 'LV':
-			if ( preg_match( '/(?:LV)?-?(\d+)/i', $postcode, $matches ) ) {
-				$postcode = count( $matches ) >= 2 ? "LV-$matches[1]" : $postcode;
-			}
+			$postcode = preg_replace( '/^(LV)?-?(.+)$/', 'LV-${2}', $postcode );
```

Comment:
Is there any reason to remove the `(\d+)` here? AFAIK the postcode is all digits, right? So not reason to capture "more than needed". The same applies to CZ/SK above.

Reply:
You're right, the postcodes are all digits. I think I got the idea for a broader regex from this newly added DK regex:
https://github.com/woocommerce/woocommerce/blob/073c3ff3b420d6cdb608a5b27cdff522a08dc8e6/plugins/woocommerce/includes/wc-formatting-functions.php#L1011-L1013
I narrowed the regex for LV, CZ and SK in a new commit and left DK as is for now.